src/vivarium/framework/components/manager.py

```diff
@@ -16,47 +19,63 @@ class ComponentConfigError(VivariumError):
 
 
 class ComponentManager:
-    """Handles the setup and access patterns for components in the system."""
+    """Handles tracking and access patterns for all components in a Vivarium
+    and initiating the ``setup`` life-cycle stage of each component."""
 
     def __init__(self):
         self._managers = []
         self._components = []
 
     def add_managers(self, managers: Sequence):
-        """Register new managers with the system.
-
-        Managers are setup before components
+        """Registers new managers with the system. Managers are setup before
+        components.
 
         Parameters
         ----------
         managers:
-          Components to register
+          Instantiated managers to register
         """
         self._add_components(self._managers, managers)
 
     def add_components(self, components: Sequence):
-        """Register new components.
+        """Registers new components with the system. Components are setup after
+        managers.
 
         Parameters
         ----------
         components:
-          Components to register
+          Instantiated components to register
         """
         self._add_components(self._components, components)
 
-    def _add_components(self, component_list, components):
+    def _add_components(self, component_list: Sequence, components: Sequence):
```

Comment:
I don't think you want sequence here - that includes strings

Reply:
yep, that's my holdout from before I knew typing very well.  ``Union[List, Tuple]`` is what we want.

Reply:
This isn't quite right, because we are ok with a singular component, but we can't really type check that :-/